code/WorkInProgress/Ported/policetape.dm

```diff
@@ -144,7 +144,7 @@
 		for(var/atom/A in T) //Check to see if there's anything solid on the tape's turf (it's possible to build on it)
 			if(A.density)
 				return
-		M:loc = T
+		M:forceMove(T)
```

Comment:
Yes, but removing `:` is a project for a future PR that will require a more complex replacer. `as` does nothing to define the var type.